wallet/lib/pages/home/currency_screen/model/ibc_coins.dart

```diff
@@ -143,9 +143,18 @@ extension IBCCoinsDePar on IBCCoins {
       case IBCCoins.ujuno:
       case IBCCoins.uatom:
       case IBCCoins.ustripeusd:
-        return (double.parse(amount) / kBigIntBase).toStringAsFixed(2);
+        final double denomAmount = double.parse(amount) / kBigIntBase;
+        if(denomAmount > 0 && denomAmount < 100){
+          return denomAmount.toStringAsFixed(2);
+        }else if(denomAmount >= 100 && denomAmount < 1000){
+          return denomAmount.toStringAsFixed(0);
+        }else if(denomAmount >= 1000){
+          final RegExp reg = RegExp(r'(\d{1,3})(?=(\d{3})+(?!\d))');
+          return (denomAmount.round()).toString().replaceAllMapped(reg, (Match match) => '${match[1]},');
+        }
+        return amount;
       case IBCCoins.upylon:
-        return (double.parse(amount) / kBigIntBase).toStringAsFixed(0);
+        return (double.parse(amount) / kBigIntBase).toStringAsFixed(2);
```

Comment:
Pylons coin will never be in decimals.

Reply:
yes I did point out this thing on Friday's Scrum meeting, @StefanB90 it is intended and he want it to be like this.

Reply:
@StefanB90 kindly provide info on this please.